contrib/sdl1.2_mixer/template.py

```diff
@@ -0,0 +1,32 @@
+pkgname = "sdl1.2_mixer"
+pkgver = "1.2.12"
+pkgrel = 0
+build_style = "gnu_configure"
+# make sure they're pulled as runtime deps
+configure_args = [
+    "--disable-music-mod-shared",
+    "--disable-music-ogg-shared",
+    "--disable-music-flac-shared",
+]
+configure_gen = []
+make_cmd = "gmake"
+hostmakedepends = ["automake", "gmake", "pkgconf"]
```

Comment:
also noticed that configure complains about missing /usr/bin/file but not sure if that's important, might as well add file to hostdeps anyway
```suggestion
hostmakedepends = ["file", "gmake", "pkgconf"]
```

Reply:
it's literally not used for anything

Reply:
/shrug, it does try to run it and that fails and complains but ok